db/data_migration/20160913172039_add_related_mainstream_record_for_detailed_guides.rb

```diff
@@ -0,0 +1,55 @@
+RelatedMainstream.destroy_all
```

Comment:
For future ref, `delete_all` will do a pure `DELETE`, whereas `destroy_all` loads each record first then deletes it, as it might have some `{before,after}_destroy` or `dependent: :destroy` logic.